### PR TITLE
Hello-rails-backend

### PR DESCRIPTION
**Pull request by emhamza**
### In this milestone I have developed a two separate apps - one for backend and second for frontend - for my hello-project. Following are the tasks that I have accomplished in the backend: 🎉 

- [x] Create a new Rails API app called 'hello-rails-back-end'.
- [x] Initialize your project with Git.
- [x] Make sure that your project has a Postgres database set up.
- [x] Create a table for storing your messages and create 5 different greetings.
- [x] Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,190 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/emhamza/ror-api-backend/blob/dev/LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<a name="readme-top"></a>
```
Comment (mikethreels) [inline README.md]: - [x] the link to the MIT license is not working as expected please make sure it is referring to the actual license.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Hamza, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good job connecting the projects

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (emhamza) [reply README.md]: Certainly, thank you for pointing that out 🎉

**Review by addod19 — APPROVED** [review]
## Hi @emhamza, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] Nice job fixing the link to the MIT link :rocket:


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**